users/tests.py

```diff
@@ -28,6 +28,18 @@ def test_guest_can_request_for_otp(self):
         otp_exists = OneTimePassword.objects.filter(mobile=self.user.mobile).exists()
         self.assertTrue(otp_exists)
 
+    def test_otp_not_passed(self):
+        # check if otp is present
+        response = self.client.post(reverse("verify-otp"), {"mobile": self.user.mobile})
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.data["detail"], "OTP Not Provided")
```

Comment:
your error messages are different. In one case, it is "not provided" and in the other case, it is "not defined". Given that the issue is the same (some parameter not being provided), the error messages should be consistent too!